### PR TITLE
[Fixed Recently Ordered sidebar error in a special character like TM]

### PR DESCRIPTION
**Pull request by hitesh-wagento**
### Description (*)
Fixed Recently Ordered sidebar error in a special character like **TM**

### Fixed Issues (if relevant)

1. magento/magento2#26426: Recently Ordered sidebar error in a special character like TM

### Manual testing scenarios (*)

1. Order placed any product
2. Go to a category page
3. You can see the left side Recently Ordered section

### Expected result

expected

### Actual result

actual

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by hitesh-wagento** [comment]
Hello @magento-engcom-team 

Is there any update in this PR ?

Thanks

File: app/code/Magento/Sales/view/frontend/templates/reorder/sidebar.phtml [hunk 1]
```diff
@@ -33,15 +34,17 @@
...
+                                        '<?= $block->escapeHtml(__('Product is not salable.')) ?>'
                                    },
                                    disable: !is_saleable"
                                    class="checkbox" data-validate='{"validate-one-checkbox-required-by-name": true}'/>
                         </div>
                     </div>
                     <strong class="product-item-name">
                         <a data-bind="attr: {href: url}" class="product-item-link">
-                            <span data-bind="text: name"></span>
+                            <span data-bind="html: name"></span>
```
Comment (dmytro-ch) [inline app/code/Magento/Sales/view/frontend/templates/reorder/sidebar.phtml:47]: Apparently this output should be escaped.
I'm not sure we should apply this change from the security perspective.

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

File: app/code/Magento/Sales/view/frontend/templates/reorder/sidebar.phtml (hunk 1 above)
Comment (hitesh-wagento) [reply app/code/Magento/Sales/view/frontend/templates/reorder/sidebar.phtml:47]: Hi @dmytro-ch  

If We will not apply this change it shows error in frontend

Thanks

**Comment by hitesh-wagento** [comment]
#MM20IN

**Comment by hitesh-wagento** [comment]
Hi @dmytro-ch, @sivaschenko 

I have updated my comment is there any update in this PR?

https://github.com/magento/magento2/pull/26428#discussion_r371596413

Thanks

**Comment by hitesh-wagento** [comment]
Hello @magento-engcom-team

Is there any update in this PR ?

Thanks

**Comment by dmytro-ch** [comment]
Hello @hitesh-wagento, sorry for the late reply.
I guess we still need to find a better solution or at least make sure this change do not introduce a potential security issue. I'll perform the investigation as soon as I can.

**Comment by dmytro-ch** [comment]
Hello @hitesh-wagento, I've rechecked the current solution and confirm that it potentially allows rendering malicious content on the page.
Screenshot 2020-05-10 at 15 41 28


A for the original issue, it's quite disputable. I think we still can use the character itself `™`, instead of its HTML code `&trade;`. It worked well for me during testing.

So, I'm closing this task due to the absence of fix necessity.

Thank you!

**Pull request closed without merging**